nc-games/src/utils/api.js

```diff
@@ -4,8 +4,24 @@ const gamesApi = axios.create({
   baseURL: "https://ashs-nc-games-project.herokuapp.com/api",
 });
 
-export const getReviews = () => {
-  return gamesApi.get(`/reviews`).then(({ data }) => {
+export const getReviews = (category) => {
+  let queryStr = "/reviews";
+  if (category) queryStr += `?category=${category}`;
+  console.log(queryStr, "< is this right?");
+  return gamesApi.get(queryStr).then(({ data }) => {
+    console.log(data, "from api");
     return data;
   });
 };
+
+export const getCategories = () => {
+  return gamesApi.get("/categories").then(({ data }) => {
+    return data.categories;
+  });
+};
+
+export const getReviewsByCategories = (category) => {
+  return gamesApi.get(`/reviews?category=${category}`).then(({ data }) => {
+    return data.reviews;
```

Comment:
to make this (getReviewsByCategory) DRYer, you could refactor to make category an optional parameter for getReviews and then check if category is defined or not before making your get request